Makefile

```diff
@@ -8,17 +8,18 @@ help:
 
 WARNINGS=RUSTDOCFLAGS="-D warnings"
 DEBUG_ASSERTIONS=RUSTFLAGS="-C debug-assertions"
+ALL_FEATURES_BUT_ASYNC=--features concurrent,default,std,testing,serde
```

Comment:
Do we need to list `default` and `std` features here? Isn't `default` enabled by default and `std` implied by `default`?

Reply:
Indeed you are right. 

Removed std and default from the var.

Reply:
We do not need to list default it's always on ( except if we use `--no-default-features`